src/sentry/tasks/assemble.py

```diff
@@ -167,10 +167,10 @@ def assemble_artifacts(org_id, version, checksum, chunks, **kwargs):
     from sentry.utils.zip import safe_extract_zip
     from sentry.models import File, Organization, Release, ReleaseFile
 
-    with configure_scope() as scope:
-        scope.set_tag("organization", org_id)
+    organization = Organization.objects.get(id=org_id)
```

Comment:
Should this be a cached get?

Reply:
its the same query that was executing before

Reply:
We've been trying to get all the repetitive DB hits out post_process to help with performance.

Reply:
Thats great, but its not related with this change unless you're asking me to remove context entirely there. Otherwise whomever is solving that can solve that in a more concise change.